### DISCUSSION TITLE
Display a blank page after logging in

### DISCUSSION THREAD

**Discussion by charliboy** [open-webui/open-webui#3268] [General]
# Bug Report
Models.svelte:24 Uncaught (in promise) TypeError: w.at is not a function
    at Models.svelte:24
    at Array.filter (<anonymous>)
    at Object.Sr.n.$$.update (Models.svelte:24)
    at Ot (Component.js:144)
    at new Dr (Models.svelte:28)
    at ko (MessageInput.svelte:322)
    at Ot (Component.js:148)
    at new Eo (MessageInput.svelte:78)
    at Sn (Chat.svelte:1395)
    at ki (Chat.svelte:1319)
## Description
Display a blank page after logging in
**Bug Summary:**
[Provide a brief but clear summary of the bug]
If the Ollama service is stopped, a page will display normally after logging in; When the Ollama service is working properly, after logging in, a blank page is displayed. Upon inspection, it is found that the page displays the errors listed above
**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

**Expected Behavior:**
[Describe what you expected to happen.]

**Actual Behavior:**
[Describe what actually happened.]

## Environment
   
- Open WebUI Version(python3.11): [ 0.3.5]
- Ollama (if applicable): [0.1.44]

- Operating System: [ Ubuntu 22.04]
- Browser (if applicable): [Chrome  91.0.4472.106]

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]
Models.svelte:24 Uncaught (in promise) TypeError: w.at is not a function
    at Models.svelte:24
    at Array.filter (<anonymous>)
    at Object.Sr.n.$$.update (Models.svelte:24)
    at Ot (Component.js:144)
    at new Dr (Models.svelte:28)
    at ko (MessageInput.svelte:322)
    at Ot (Component.js:148)
    at new Eo (MessageInput.svelte:78)
    at Sn (Chat.svelte:1395)
    at ki (Chat.svelte:1319)
**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method
pip install open-webui
[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by FHY07** [comment]
Hello, I also encountered this bug, how can I solve it?

**Reply by rgaricano** [reply]
are you tried to clean cache & site cookies?
